chapters/interfaces.txt

```diff
@@ -989,7 +989,7 @@ ifdef::VK_VERSION_1_2,VK_EXT_scalar_block_layout[]
   member must: be aligned according to its scalar alignment.
 endif::VK_VERSION_1_2,VK_EXT_scalar_block_layout[]
 ifdef::VK_VERSION_1_1,VK_KHR_relaxed_block_layout[]
-. All vectors must: be aligned according to their scalar alignment.
+. All vectors must: be aligned according to their scalar alignment and must: not improperly straddle, as defined above.
```

Comment:
This section is only really defining the alignment of the member, and so this doesn't really fit in here. It's confusing because the alignment rules are nearly all the rules, so the rest are easy to miss. Perhaps moving the other rules ahead of the alignment rules instead of this change would make it less confusing?

Reply:
Ok, I see. Moving ahead doesn't change a lot because the sentense "All vectors must be aligned according to their scalar alignment." sounds like and absolute rule. I thought about dividing rules into two parts a) simple alignment rule for the case when scalarBlockLayout  is enabled. b) all other cases. This could simplify things a bit because in b) we will have 3 rules instead of 4 and scalarBlockLayout will be mentioned only once. From that point it would be possible to move further but actually I don't see a good way how to fix this completely, so I close this.